dynamic_rest/viewsets.py

```diff
@@ -582,8 +582,18 @@ def update(self, request, *args, **kwargs):
                     return self._bulk_update(bulk_payload, partial)
 
         # singular update
-        return super(DynamicModelViewSet, self).update(request, *args,
-                                                       **kwargs)
+        try:
+            return super(DynamicModelViewSet, self).update(request, *args,
+                                                           **kwargs)
+        except AssertionError as e:
+            if 'lookup_field' in str(e):
```

Comment:
Where are these errors raised? Wondering if this is a safe assumption, and/or if the errors is from within DREST we could raise a custom error (but maybe it's from DRF/Django?)

Reply:
@ryochiji I have updated the check to use a more reliable substring of the DRF assertion, changed the error message of the 400 to be more descriptive, and left a comment about the reasoning behind this change (DREST's support for bulk operations means that an update-without-ID should NOT be considered a misconfiguration as it is in DRF)